architecture/alpha.md

```diff
@@ -1,11 +1,163 @@
 
+## Table of Contents
+
+1. Overview
+  - Who is this for?
+  - What values does this project follow?
+  - What principles does this project follow?
+1. Scope
+1. Value
+1. Risks & Rabbit Holes
+1. Appetite
+
 ## Overview
 
+`ekipse` is the codename for this project.
+
+### Who is this for?
+
+This project is intended to create a standards-based website builder for:
+
+- Small personal websites (1-3 content types, less than 50-100 pages)
+  - Blogs
+  - Portfolios
+  - Developer blogs
+  - Et cetera
+- Landing pages
+  - Marketing pages
+  - Ad landing pages
+  - Et cetera
+
+### What values does this project follow?
+
+The `ekipse` project and maintainers align with the following values:
+
+- **Antifragility:** the `ekipse` project should not be fragile. It should live on, as the market changes, without requiring work from its users.
+- **Simplicity**: the `ekipse` project does not do everything. It meets the needs of the cohorts it is built for, and nothing more. Every feature we choose to not add, or choose to remove, is a win for the project.
+- **Education**: the `ekipse` project should be a tool for learning the web. It shouldn't introduce enterprise abstractions to solve complexity that the cohorts of this project doesn't have.
+
+### What principles does this project follow?
+
+The `ekipse` project and maintainers align with the following principles, which are in accordance with our values:
+
+- **Closed to extension**: 
+  - It should be closed to extension, to prevent dependency hell. 
+  - Advanced features are solved out of `ekipse`, not inside it.
+  - _This will ensure a project built with `ekipse` today, works next year._
+- **Evergreen development**: 
+  - It should be built on web standards, so that the tooling never dies.
+  - It should handle changing tides without breaking. `ekipse` relies on as few dependencies as possible, and the user never manages those dependencies.
+  - It should be designed to allow changing dependencies, without the user noticing an API change. We deal with the pain of dependency management, not our users. 
+- **Documentation over code contribution**:
+  - Problems solved outside of `ekipse`'s wheelhouse with `ekipse` should not become code; they should become content.
+- **Come as you are, and grow beyond**:
+  - `ekipse` asks people to come as they are, and write simple web standard code to build a website.
+  - When someone's ability grows beyond `ekipse`'s built-in capabilities, we offer an escape hatch, instead of leaving them hanging out to dry.
+
 ## Scope
 
+For the purposes of the alpha, `ekipse` is/will:
+
+- Closed to modification and extension, but open to ejection or forking
+- Built on modern web standards, no pre-compiling:
+  - ES6 JS, polyfilled with `esbuild` according to a `browserslist` config in `package.json`
+  - Modern CSS, polyfilled with `postcss` and `postcss-preset-env`
+  - Modern HTML, with templating powered under the hood by `reshape` and `reshape-standards`
```

Comment:
`reshape-standard`  may be replaced with something simpler, right now it uncludes a few plugins that may be considered out of scope for the project, namely: `sugarml` parser, `reshape-content` (for `.md`), and `reshape-retext`

Reply:
```suggestion
  - Modern HTML, with templating powered under the hood by `reshape` and a few built-in reshape plugins for templating
  - expressions
  - data injection via a reserved constant
  - one-way includes w/ variables
```